django_project/base/models/project.py

```diff
@@ -208,3 +215,14 @@ def pagination_threshold_exceeded(self):
             return True
         else:
             return False
+
+
+class ProjectScreenshots(models.Model):
```

Comment:
I think the name should be singular - ProjectScreenshot (and for description below)